client/lib/signup/site-type.js

```diff
@@ -165,6 +165,7 @@ export function getAllSiteTypes() {
 			siteTopicLabel: i18n.translate( 'What type of products do you sell?' ),
 			customerType: 'business',
 			purchaseRequired: true,
+			forcePublicSite: true,
```

Comment:
Can we leave a comment here as a warning for other devs until we can get some e2e's in place?